### DISCUSSION TITLE
FPGA or some specialized hardware for running 405b cheaply?

### DISCUSSION THREAD

**Discussion by sprappcom** [ggml-org/llama.cpp#9087] [Q&A]
Does anyone know of any solutions to run 405b cheaply? I understand some people suggest P40 etc but it's quite expensive in terms of running cost as each 24GB requires 250W.

the "cheapest" i can think of is.
1. amd epyc 12 channel with 460GB/s ram
2. clustering jetson nano

these 2 for long running power consumption savings. what about u guys? i was thinking some FPGA solution or something now

**Comment by disdi** [comment]
I dont think this project supports any FPGA backend as of now .
It would be very interesting to have one.
